### PR TITLE
Beta 0.1.1 update.

### PR DESCRIPTION
**Pull request by mrbusysky**
Added new features to inpainting.
* Outpainting
* Crop
* Load mask from file
* Ui updates

Added a new editor window Called ImageEditor.
* Drawing with color
* Create image
* Crop image

Edited promptwindow.
* Edited Api for new remove background calls
* Ui a bit for buttons
* Modified post method for outpainting

Updated thumbnail method for models.

Updated the + count to 3 in promptwindow.

and much more.

### CODE REVIEW

**Review by nbrucker — APPROVED** [review]
lgtm

**Comment by mrbusysky** [comment]
Okay model and such has been updated

File: Scenario/Editor/Models/ModelsUI.cs [hunk 1]
```diff
@@ -171,25 +173,45 @@ public void OnGUI(Rect position)
...
+            string downloadUrl = null;
+            
+            if (item.trainingImages != null && item.trainingImages.Count > 0)
+            {
+                downloadUrl = item.trainingImages[0].downloadUrl;
+            }
+            else if (item.thumbnail != null && !string.IsNullOrEmpty(item.thumbnail.url))
+            {
+                downloadUrl = item.thumbnail.url;
+            }
```
Comment (nbrucker) [inline Scenario/Editor/Models/ModelsUI.cs:208]: I think we want it to be the other way around.
The thumbnail comes first, then the training image
See method in webapp
image

**Comment by mrbusysky** [comment]
Okay, it has been switched.

**Review by nbrucker — APPROVED** [review]

**Comment by mrbusysky** [comment]
@nbrucker You should be able to merge now

**Review by hervenivon — APPROVED** [review]
LGTM

**Pull request merged**